actions/websocket_actions.jsx

```diff
@@ -521,24 +523,24 @@ export function handleEvent(msg) {
         break;
 
     case SocketEvents.SIDEBAR_CATEGORY_CREATED:
-        dispatch(handleSidebarCategoryCreated(msg));
+        wrapEventWithJitter(handleSidebarCategoryCreated, msg, dispatch);
         break;
 
     case SocketEvents.SIDEBAR_CATEGORY_UPDATED:
-        dispatch(handleSidebarCategoryUpdated(msg));
+        wrapEventWithJitter(handleSidebarCategoryUpdated, msg, dispatch);
         break;
 
     case SocketEvents.SIDEBAR_CATEGORY_DELETED:
-        dispatch(handleSidebarCategoryDeleted(msg));
+        wrapEventWithJitter(handleSidebarCategoryDeleted, msg, dispatch);
```

Comment:
Regarding what I mentioned on Community, I don't think we should have jitter for these since they're almost always affecting just the current user. This also might make the issue where you double click the category header behaves oddly (https://github.com/mattermost/mattermost-server/issues/22044) worse by increasing that delay

Reply:
removed